tests/qa_automation/patch_and_reboot.pm

```diff
@@ -63,6 +63,11 @@ sub run {
     upload_logs('/tmp/kernel_changelog.log');
     upload_logs('/tmp/repos_list.txt');
 
+    if (get_var('SAVE_LIST_OF_PACKAGES')) {
+        assert_script_run("rpm -qa > /tmp/rpm_packages_list.txt");
```

Comment:
why not to have `before` or `after` in the file name?

Reply:
Will fix that. Thanks.